src/main/java/com/google/sps/servlets/ProfileRoutes.java

```diff
@@ -56,15 +66,21 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         Map<Key, Entity> routesList = datastore.get(routesKeys);
 
         Route newRoute;
+        RouteWithType newRouteWithType;
+        int i = 0;
```

Comment:
Is this variable used afterwards?

Reply:
Yeah, I used it to set userAccess for each route, but in next commit I'll add new property to the Route class so it won't be used anymore.